zend_abstract_interface/symbols/lookup.c

```diff
@@ -142,16 +120,9 @@ static inline zend_class_entry *zai_symbol_lookup_class_impl(zai_symbol_scope_t
         } break;
 
         case ZAI_SYMBOL_SCOPE_NAMESPACE: {
-            zai_str key = zai_symbol_lookup_key(scope, name, true);
-
-            if (key.ptr) {
-                result = zai_symbol_lookup_table(
-                    EG(class_table),
-                    key,
-                    false, true);
-
-                pefree((char*) key.ptr, 1);
-            }
+            zai_string key = zai_symbol_lookup_key(scope, name, true);
```

Comment:
It feels strange to create / destroy a memory space at every lookup for concat, though I guess it would pop up if we ever need to optimize this.